### PR TITLE
PEM 9.8 - release branch

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Added PEM 9.8 release notes

### CODE REVIEW

File: product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Released: 19th Nov 2024
+
+New features, enhancements, bug fixes, and other changes in PEM 9.8.0 include:
+
+| Type         | Description                                                                                                                                               |
+|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enhancement  | Added support for EDB Advanced Server, EDB Postgres Distributed, and PostgreSQL 17.                                                                       |
+| Enhancement  | Added support for RHEL9 ARM64 platform.                                                                                                                   |
+| Enhancement  | Added support to allow sslmode connections from PEM agent to PEM server.                                                                                  |
```
Comment (sonotley) [inline product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx:14]: Do you know if we have updated docs for this @nidhibhammar?

File: product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+Released: 19th Nov 2024
+
+New features, enhancements, bug fixes, and other changes in PEM 9.8.0 include:
+
+| Type         | Description                                                                                                                                               |
+|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enhancement  | Added support for EDB Advanced Server, EDB Postgres Distributed, and PostgreSQL 17.                                                                       |
+| Enhancement  | Added support for RHEL9 ARM64 platform.                                                                                                                   |
```
Comment (sonotley) [inline product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx:13]: Need to remember to update the Postgres version matrix for this change.

**Review by sonotley — CHANGES_REQUESTED** [review]
Suggested some changes and additions, plus a few reminders for other areas of the docs

File: product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx:19]: Added this to registering agent topic

File: product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx (hunk 2 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/9/pem_rel_notes/980_rel_notes.mdx:18]: Platform compatibility page is updated by website team

**Review by sonotley — APPROVED** [review]
LGTM - will follow up later this week to see if we need to add something more for sslmode but let's get this PR merged.

**Pull request merged**